framework/src/modules/interoperability/mainchain/store.ts

```diff
@@ -161,7 +161,8 @@ export class MainchainInteroperabilityStore extends BaseInteroperabilityStore {
 		// Chain must be live; This check is always on the receivingChainID
 		const isReceivingChainLive = await this.isLive(
 			receivingChainIDAsStoreKey,
-			sendContext.timestamp,
+			// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+			sendContext.timestamp!,
 		);
```

Comment:
maybe its better to check if timestamp is undefined and throw error in isLive function in mainchain as its needed to check for the liveness condition

Reply:
isLive is being called from other places too, so maybe its better to throw it from one place

Reply:
other places don't have optional param,  no ?

Reply:
ah true, sorry ignore my comment